airbyte-integrations/connectors/destination-azure-blob-storage/src/test-integration/java/io/airbyte/integrations/destination/azure_blob_storage/AzureBlobStorageCsvDestinationAcceptanceTest.java

```diff
@@ -103,4 +107,29 @@ private static JsonNode getJsonNode(final Map<String, String> input, final Map<S
     return jsonRecords;
   }
 
+  @Override
+  protected String getAllSyncedObjects(String streamName) {
+    try {
+      final List<AppendBlobClient> appendBlobClients = getAppendBlobClient(streamName);
+      StringBuilder result = new StringBuilder();
+      for (AppendBlobClient appendBlobClient : appendBlobClients) {
+        if (result.isEmpty()) {
+          final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+          appendBlobClient.download(outputStream);
+          result.append(outputStream.toString(StandardCharsets.UTF_8));
+        } else {
+          final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+          appendBlobClient.download(outputStream);
+          var stringStream = outputStream.toString(StandardCharsets.UTF_8);
+          result.append(stringStream.substring(stringStream.indexOf('\n') + 1));
+        }
+      }
+      LOGGER.info("All objects: " + result);
+      return result.toString();
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Does it make sense to call `LOGGER.error` here?

Reply:
updated it